index.yaml

```diff
@@ -837,10 +876,10 @@ entries:
       catalog.cattle.io/ui-component: rancher-cis-benchmark
     apiVersion: v1
     appVersion: v1.0.4
-    created: "2021-06-11T16:24:24.721891-07:00"
+    created: "2021-06-17T10:22:59.635631-07:00"
     description: The cis-operator enables running CIS benchmark security scans on
       a kubernetes cluster
-    digest: 72d5e03b9b75c36507ba56bbe762baddca7348167bf2b811a57b21810f2be064
+    digest: ef7c8837f09bcf669bcb7aef5aec2dca84a27e2c93da5b44194fb0113a1c527d
```

Comment:
This is the correct value for this version, and I see the same value in the release-v2.5 branch here: https://github.com/rancher/charts/blob/037989f43e8090a362f86e3ad6a21c981337a0f8/index.yaml#L818

Somehow the index.yaml file was compromised during development in the dev-v2.5 branch.